test/python/test_ttir_ops.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# RUN: %python %s | FileCheck %s
+
+import sys, os
+from test_infra.ttir_builder import TTIRBuilder, compile_as_mlir_module, Operand
+import logging
+
+
+@compile_as_mlir_module((32, 32), (32, 32))
+def test_add(in0: Operand, in1: Operand, builder: TTIRBuilder):
+    return builder.add(in0, in1)
+
+    # CHECK: %0 = tensor.empty() : tensor<32x32xf32>
+    # CHECK: %1 = "ttir.add"(%arg0, %arg1, %0)
+    # CHECK: return %1 : tensor<32x32xf32>
```

Comment:
I think it's a bit more readable to move these above the `return`.  It's more in line with the style of the mlir unit tests.